osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -229,6 +256,13 @@ protected virtual void BindProcessor(ScoreProcessor processor)
             ComboCounter?.Current.BindTo(processor.Combo);
             HealthDisplay?.Current.BindTo(processor.Health);
 
+            processor.NewJudgement += _ =>
+            {
+                processor.PopulateScore(score);
+
+                PerformanceCounter.Current.Value = performanceCalculator.Calculate(performanceClock.CurrentTime);
```

Comment:
No null check, `performanceCalculator` can be null if it is not yet implemented for the current mode.